### DISCUSSION TITLE
Does TeslaUSB Archive all DashCam Footage?

### DISCUSSION THREAD

**Discussion by The-Gremlee** [marcone/teslausb#810] [Q&A] [answered]
I have a 128GB SSD in a Raspberrry Pi Zero W and it works a treat.  When my model Y gets home, it archives all the sentry clips but few of the dashcam videos.  I set the RecentClips size to 30G to begin with, and then amended it to 60G.  When my car gets home it only seems to archive the most recent clip.  We don't tend to travel far from home so the recentclips, which is set to archive, will rarely be more than 30 minutes.
Am I doing something wrong, or is this capability not there?

**Comment by marcone** [comment]
Where do you live? In some countries the car doesn't keep a full hour of RecentClips, and you'll need to adjust the `SNAPSHOT_INTERVAL` variable accordingly.

**Reply by The-Gremlee** [reply]
I live in the UK (specifically England).  What should I update the SNAPSHOT_INTERVAL setting to?

**Reply by marcone** [reply, marked as answer]
I'm not sure what Tesla does in the UK, but try uncommenting [this](https://github.com/marcone/teslausb/blob/c27e8e905c29750c9ab1cbe684b2733e5347c4b4/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample#L61) line in the config.

**Reply by The-Gremlee** [reply]
I will do.  Thank you.
